src/components/playground/runtime.js

```diff
@@ -44,9 +46,11 @@ export class Runtime {
         throw new Error('Missing render function on default export');
 
       const userOutput = render();
+      // This limiter setting preserves approx. 1 dB of headroom.
+      const limit = (input) => el.compress(1, 10, -4, 20, input, input);
```

Comment:
The compress function that we're using right here comes from the `el` library imported from `@elemaudio/core` version at the top of the file. But the `userOutput` signal that we get from invoking the user module is built using the `el` export of whatever version `@elemaudio/core` is loaded in the user module.

It's really quite rare that there would be a conflict between these versions, but these versions _do_ sometimes have requirements onto the renderer. For example, the core library might export a utility that uses a native node that only a recent version of the renderer knows about.

I do anticipate the next major version bringing some breaking changes, so we could arrive in a situation here where the user code is running a v4 web-renderer instance and v4 elemaudio/core but the limiter here is written against v3 elemaudio/core, and we could be getting into tricky territory there.

Even if this wouldn't be a problem, I think it'd be safer to make sure that we try to use the `el.compress` function from the same version that the user is running. Luckily, we know exactly what version that is because we force it using the importMap script above. Then on line 16 we fetch a matching web-renderer version. I think the fix here would be to add another line right below 16 which fetches the appropriate `elemaudio/core` version and grabs the `el` library out of that, then uses that compress function here to apply the limiting

Reply:
Thanks for getting back, @nick-thompson. I see your concern, and I totally agree that keeping the versions aligned on import is a much better solution. I'll try to knock out the changes this weekend.